tests/spread/general/license-valid/task.yaml

```diff
@@ -7,20 +7,20 @@ prepare: |
   mkdir test-snap
   cd test-snap
   snapcraft init
-  
-  # License is currently base triggered only
-  if [[ "$SPREAD_SYSTEM" =~ ubuntu-18.04 ]]; then
-      echo "base: core18" >> snap/snapcraft.yaml
-  elif [[ "$SPREAD_SYSTEM" =~ ubuntu-16.04 ]]; then
-      echo "base: core16" >> snap/snapcraft.yaml
-  else
-      FAIL "Test not supported for $SPREAD_SYSTEM"
-  fi 
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "../snaps/$SNAP/snap/snapcraft.yaml"
```

Comment:
the setup runs "snapcraft init" so the leading ../ is wrong

Reply:
Fixed!